checker/src/main/java/org/checkerframework/checker/i18n/qual/Localized.java

```diff
@@ -16,21 +16,20 @@
  * @checker_framework.manual #i18n-checker Internationalization Checker
  */
 @SubtypeOf(UnknownLocalized.class)
-@ImplicitFor(
-        literals = {
-            /* All integer literals */
-            LiteralKind.INT,
-            LiteralKind.LONG,
-            LiteralKind.FLOAT,
-            LiteralKind.DOUBLE,
-            LiteralKind.BOOLEAN,
+@QualifierForLiterals({
+    /* All integer literals */
```

Comment:
This comment is confusing, as below it includes floats and boolean.
I see that this is from before, but let's also clean this up.
I think it would be better to explicitly say that these are all primitives other than char and string, because the latter two might require localization.